src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/Ordering/AbstractAddItemCommand.cs

```diff
@@ -56,9 +61,9 @@ protected override async Task<bool> TryHandleCommandAsync(IProjectTree node, boo
             return true;
         }
 
-        private IProjectTree GetNodeToAddTo(IProjectTree node)
+        private IProjectTree? GetNodeToAddTo(IProjectTree node)
```

Comment:
This looks odd.. I guess its because `node.Parent` is nullable?

Reply:
Yes.